openhands/runtime/plugins/agent_skills/utils/aider/linter.py

```diff
@@ -151,11 +199,17 @@ def basic_lint(fname, code):
     errors = traverse_tree(tree.root_node)
     if not errors:
         return
-    return LintResult(text=f'{fname}:{errors[0]}', lines=errors)
+
+    error_messages = [
+        f'{fname}:{line}:{col}: {error_details}' for line, col, error_details in errors
+    ]
+    return LintResult(
+        text='\n'.join(error_messages), lines=[line for line, _, _ in errors]
+    )
 
 
 def extract_error_line_from(lint_error):
-    # moved from openhands.agentskills#_lint_file
+    first_error_line = 0
```

Comment:
To ensure the behavior is consistent (since we use `None` everywhere else)

```suggestion
    first_error_line = None
```

Reply:
Will change that, thanks